src/context/Reducer/WishListItemReducer.js

```diff
@@ -0,0 +1,16 @@
+export const WishListItemReducer=(state,action)=>{
+    const {wishListItem,wishListCount}=state;
+    const {payload}=action;
```

Comment:
You can destructure `type`here

Reply:
Will do. But what happened I forgot to commit some changes before ,that is why this file and commits are too big